acceptance-tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Stories/ServiceSearchTests.cs

```diff
@@ -54,18 +53,39 @@ public void Subsequent_Filter_Postcode_Search_As_A_Connect_User()
            .Then(s => _steps.Then200StatusCodeReturned())
            .BDDfy();
    }
-  
-  
-   //Negative Scenarios
    [TestMethod]
-   public void No_Metrics_Are_Sent_When_A_Non_200_Message_Is_Received_From_Postcode_IO()
+   public void Initial_Postcode_Search_As_A_Find_User_When_ServiceSearchAPI_Responds_With_A_Non_200_ResponseCode()
    {
-       this.Given(s => _steps.GivenIHaveASearchServiceRequest("15","E1 2EN","400","1","1"))
+       this.Given(s => _steps.GivenIHaveASearchServiceRequest("15","E1 2EN","300","1","2"))
            .When(s => _steps.WhenISendARequest())
-           .Then(s => _steps.Then500StatusCodeReturned())
+           .Then(s => _steps.Then200StatusCodeReturned())
            .BDDfy();
    }
-
+   [TestMethod]
+   public void Initial_Postcode_Search_As_A_Connect_User_When_ServiceSearchAPI_Responds_With_A_Non_200_ResponseCode()
+   {
+       this.Given(s => _steps.GivenIHaveASearchServiceRequest("15","E1 2EN","500","1","1"))
+           .When(s => _steps.WhenISendARequest())
+           .Then(s => _steps.Then200StatusCodeReturned())
```

Comment:
question: This test is a bit confusing, the title says that it should respond with a non-200 response code, but the test steps are testing that a 200 response code is returned?